### DISCUSSION TITLE
Github Account Recovery - Urgent

### DISCUSSION THREAD

**Discussion by MobeenT1** [community/community#113333] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I lost access to my old github account. 2FA is enabled but i lost my phone. Recovery code are not working. I am not able to login. 
I have submitted ticket on github support. Its been 2 days no response. How can I recovery my account?
How long support team will take to response?

**Comment by deka27** [comment]
Don't worry they will get back to you, generally they take 48-72 hours to reply. I hope you have provided them all the necessary details so they authenticate easily.

Good luck

**Reply by asmawi117** [reply]
48-72 hours really ? i've been waiting for 35 days 😂

**Reply by MobeenT1** [reply]
35 days too long. Is there any other way to get response earlier?

**Comment by Aungkhant884** [comment]
thank

**Comment by Aungkhant884** [comment]
[Device-Info-Report (1).pdf](https://github.com/community/community/files/14670646/Device-Info-Report.1.pdf)

**Reply by MobeenT1** [reply]
?

**Comment by MobeenT1** [comment]
No response from github support team. Please help.
